powerbi-docs/developer/visuals/local-storage.md

```diff
@@ -31,22 +31,30 @@ export class Visual implements IVisual {
             this.storage = options.host.storageService;
             // ...
 
-            this.storage.get(this.updateCountName).then(count =>
-            {
-                this.updateCount = +count;
-            })
-            .catch(() =>
-            {
-                this.updateCount = 0;
-                this.storage.set(this.updateCountName, this.updateCount.toString());
+            this.storage.enabled().then(enabled => {
```

Comment:
we should document this API. 'storage.enabled'

Reply:
Added